csit-sut/Dockerfile

```diff
@@ -116,12 +109,14 @@ RUN pip install \
         scp==0.10.2 \
         six==1.12.0 \
         dill==0.2.8.2 \
-        numpy==1.14.5 \
-        scipy==1.1.0
+        numpy==1.14.5
 
-# VPP PIP pre-cache
-RUN pip install \
-        aenum
+# ARM workaround
+RUN pip install scipy==1.1.0
+
+# Kernel image for VM tests download and extract
```

Comment:
Remove

Reply:
Right, we don't need the kernel image in csit-sut when it's on the host and accessible via mount point. I'll remove it.